cmd/bundle.rb

```diff
@@ -59,7 +59,7 @@ class BundleCmd < AbstractCommand
           Add entries to your `Brewfile`. Adds formulae by default. Use `--cask`, `--tap`, `--whalebrew` or `--vscode` to add the corresponding entry instead.
 
           `brew bundle remove` <name> [...]:
-          Remove entries that match `name` from your `Brewfile`. Use `--formula`, `--cask`, `--tap`, `--mas`, `--whalebrew` or `--vscode` to remove only entries of the corresponding type.
+          Remove entries that match `name` from your `Brewfile`. Use `--formula`, `--cask`, `--tap`, `--mas`, `--whalebrew` or `--vscode` to remove only entries of the corresponding type. Passing `--formula` also removes matches against aliases and old formula names.
```

Comment:
Why only with `--formula`? Feels like it's less useful if that's required.

Reply:
Omitting all the `--*` type flags will match against all entry types (i.e. including `cask`, `mas`, etc), so it's not clear that the user specifically means a formula when omitted. `remove` is also destructive, so it's probably safer to make sure the user is clear about intent before adding a bunch of alternative names that the user may or may not have intended.

Reply:
Ok, I see. Would suggest something like:
- if `brew bundle remove` doesn't match any names but does match a valid alias/oldname: print out a warning suggesting using `brew bundle remove --formula` to remove if desired

This should help discoverability of this and generally be a bit more useful. Thoughts?

Reply:
Ok, tried this out in df78c89710692a144e5f3da9ab095aea0019f2b0. Need to add tests though.

Reply:
What's the behaviour if an alias is already specified in the Brewfile?

```rb
brew "openssl"
```

Seems like `brew bundle remove openssl` should "just work" in that scenario.

Reply:
It will just work. Just that doing `brew bundle remove --formula openssl` will now also work for
```ruby
brew "openssl@3"
```